test/testkit/verifiers/traces.go

```diff
@@ -71,3 +71,18 @@ func TracesShouldBeDelivered(proxyClient *apiserver.ProxyClient, telemetryExport
 		g.Expect(err).NotTo(gomega.HaveOccurred())
 	}, periodic.EventuallyTimeout, periodic.TelemetryInterval).Should(gomega.Succeed())
 }
+
+func TracesShouldNotBePresent(proxyClient *apiserver.ProxyClient, telemetryExportURL string, traceID pcommon.TraceID, spanIDs []pcommon.SpanID, attrs pcommon.Map, resAttrs pcommon.Map) {
+	gomega.Eventually(func(g gomega.Gomega) {
```

Comment:
I think we should use `Consistently` for this check. This implementation has the risk that we already succeed before anything arrives in the mock backend.

Reply:
changed

Reply:
It's still an `Eventually` block. Maybe you forgot to push this?